pages/Getting Started/Master-Tutorial.md

```diff
@@ -47,7 +47,15 @@ Please bear in mind 3D acceleration in VMs may be pretty slow.
 
 Hyprland can be executed by typing `Hyprland` in your tty.
 
-Systemd users can also start Hyprland, by typing `systemctl --user start hyprland-session.service` in tty. This method provides additional features, such as [xdg-autostart](https://www.freedesktop.org/software/systemd/man/latest/systemd-xdg-autostart-generator.html) support and ability to enable services for programs that rely on a graphical session and provide such services (e.g waybar).
+Systemd users can also start Hyprland, using [uwsm](https://github.com/Vladimir-csp/uwsm), either by adding the following command in shell profile.
+
+```
+if uwsm check may-start && uwsm select; then
```

Comment:
This example gives selection (whiptail) and ability to opt out to shell. It is perfectly fine, but there are shorter variants that would launch just Hyprland. I.e.:
```
if uwsm check may-start; then
    exec uwsm start hyprland.desktop
fi
```

Reply:
Didn't know about that. I'm going to test it with my shell

Reply:
Is this present in uwsm's readme? I was looking for a way to use uwsm with greetd autologin.

Reply:
https://github.com/Vladimir-csp/uwsm/blob/master/README.md#where-to-launch-from

Reply:
>  I was looking for a way to use uwsm with greetd autologin.

For autologin you can probably use `uwsm start ...` command directly.